jscomp/others/dom.ml

```diff
@@ -203,6 +209,7 @@ type _htmlVideoElement
 type htmlVideoElement = _htmlVideoElement htmlElement_like
 type location
 type window
+type windowProxy
```

Comment:
why do you need `windowProxy`? According to [MDN](https://developer.mozilla.org/en-US/docs/Glossary/WindowProxy) it's only available in a "browsing context"

Reply:
I was reading the MDN page of [MessageEvent.source](https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/source), and it seems that the source can be one of a WindowProxy, MessagePort or a ServiceWorker, so I added it.

After reading the page you linked above, I think it might be totally fine if we use window for all windowproxy...